jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt

```diff
@@ -190,3 +185,11 @@ class ExecuteCommandInShellAction(
             pluginIsInstalledAndEnabled("org.jetbrains.plugins.terminal") && AwsToolkit.isEcsExecEnabled()
     }
 }
+
+object CurrentConnectionSettings {
```

Comment:
We should make this an extension method on the AwsConnectionManager and probably move it to that file as well